lib/widgets/message_list.dart

```diff
@@ -345,7 +345,7 @@ class MessageListAppBarTitle extends StatelessWidget {
             padding: const EdgeInsetsDirectional.only(start: 4),
```

Comment:
commit message nit: it would be helpful to mention `title` in the commit summary, i.e.:

```
theme: Replace `colorMessageHeaderIconInteractive` with `title`
```